Pkgfile

```diff
@@ -3,10 +3,10 @@
 format: v1alpha2
 
 vars:
-  # renovate: datasource=git-refs versioning=git depName=https://github.com/raspberrypi/firmware.git
-  raspberrypi_firmware_ref: c276eae28a20ac7d36c533fe3a120f7679690882
-  raspberrypi_firmware_sha256: 28ca93effaea99fe2b5cd0e5daf02b30767dc77f8ce395457139766be4ac6917
-  raspberrypi_firmware_sha512: 500ba9bc5afb9e921b68c2c7642e8bd133470241cfb358df9cf9a87c6796b87c312980fc15105f95af811df3b1950324a24fb43cd34b765e41653af382daa97d
+  # renovate: datasource=github-tags depName=raspberrypi/firmware
+  raspberrypi_firmware_version: 1.20241126
+  raspberrypi_firmware_sha256: 020dbcdbb30af5942a62fc3eb355449aba45276b67e864dee2522ff53fd936e6
+  raspberrypi_firmware_sha512: 7860a7a913a35313031e07295a2c8b40b6f0e97523abf8053f0d792106dde53332755c7ab3bbe2a08b7817f77b11242637c1b0520bfb6d2d58642f6e9c4d1a52
 
   # renovate: datasource=github-tags extractVersion=^v(?<version>.*)$ depName=u-boot/u-boot
   uboot_version: 2024.07
```

Comment:
There's u-boot 2024.10 already. Does it work for Pi 4?

Reply:
don;t want to update stuff for sbc's since we know old one works

Reply:
well yeah, it's boot-time only and if it boots fine in all cases we're good